### DISCUSSION TITLE
2 questions | How to use Kiwix

### DISCUSSION THREAD

**Discussion by playerblest** [openzim/zim-requests#336] [Q&A] [answered]
Two questions please:
1. Is there a Kiwix Shortcut key to return to the previous page?
2. When I do a full text search it shows me only the first 25 results. Do you know such a problem?

Thank you!

**Comment by Popolechien** [comment, marked as answer]
Hello 

> Is there a Kiwix Shortcut key to return to the previous page?

On which platform? android is not the same as windows as apple, etc. Normally the "back" key that is specific to each platform works as intended.

> When I do a full text search it shows me only the first 25 results. Do you know such a problem?

Again the answer will be platform-specific, maybe you should open your tickets directly there (for [android](../../../kiwix/kiwix-android/issues), [apple](../../../kiwix/apple/issues), [windows/linux](../../../kiwix/kiwix-desktop/issues))?

**Reply by playerblest** [reply]
I mean the windows.
I didn't understand what you were doing "Again this is platform-specific," - I would be happy to expand.
Thanks for treatment

**Reply by playerblest** [reply]
now I get it... ok
